api/database/seeders/SkillSeeder.php

```diff
@@ -43,2324 +18,3718 @@ public function run()
+                        "fr" =>
+                        "Se remettre rapidement d’une adversité, de défis ou de changements importants ou durables.",
+                    ],
+                    "keywords" => [
+                        "en" => [
+                            "Flexibility",
+                            "Adaptability",
+                            "Adjustability",
+                        ],
+                        "fr" => ["Flexibilité", "Adaptabilité", "Ajustement"],
+                    ],
+                    "category" => "BEHAVIOURAL",
+                ],
+                "skill_family_keys" => ["personal"],
             ],
             [
-                'skill_name_en' => 'Analysis',
-                'skill_family_name_en' => 'Thinking',
+                "model" => [
+                    "key" => "resource_management",
+                    "name" => [
+                        "en" => "Resource Management",
+                        "fr" => "Gestion des ressources",
+                    ],
+                    "description" => [
+                        "en" =>
+                        "Effectively develop and allocate assets as they are needed.",
+                        "fr" =>
+                        "Développer et allouer efficacement les actifs en fonction des besoins.",
+                    ],
+                    "keywords" => [
+                        "en" => [
+                            "Asset Management",
+                            "Administration of Resources",
+                            "Resource Allocation",
+                        ],
+                        "fr" => [
+                            "Gestion des actifs",
+                            "Administration des ressources",
+                            "Allocation des ressources",
+                        ],
+                    ],
+                    "category" => "BEHAVIOURAL",
+                ],
+                "skill_family_keys" => ["leadership"],
+            ],
+            [
+                "model" => [
+                    "key" => "respect",
+                    "name" => ["en" => "Respect", "fr" => "Respect"],
+                    "description" => [
+                        "en" =>
+                        "Commit to creating an environment that is free from discrimination, harassment, and bullying.",
+                        "fr" =>
+                        "S’engager à créer un environnement exempt de discrimination, de harcèlement et d’intimidation.",
+                    ],
+                    "keywords" => [
+                        "en" => [
+                            "Consideration",
+                            "Appreciation",
+                            "Professional",
+                            "Courteous",
+                        ],
+                        "fr" => [
+                            "Considération",
```

Comment:
Something to clean-up

Reply:
`SkillSeeder` otherwise matches my results from above 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/77636306-8e27-4cf9-86fb-aa762a85ae61)

Reply:
You're looking at the SQL query?  We wanted to leave it in as a comment to make it easier to recreate the seeder again in the future if necessary.  We could leave it in a separate SQL or readme file or something if you prefer, though.

Reply:
Oh, if it is intentional then sure leaving it in is fine for reference. But I think somewhere a bit easier to find might help like a readme.